tools/devtool

```diff
@@ -529,7 +529,7 @@ ensure_ci_artifacts() {
 
     # Fetch all the artifacts so they are local
     say "Fetching CI artifacts from S3"
-    S3_URL=s3://spec.ccfc.min/firecracker-ci/v1.9/$(uname -m)
+    S3_URL=s3://spec.ccfc.min/firecracker-ci/v1.10/$(uname -m)
```

Comment:
mh, the bucket still has 4.14 guest kernels. Shouldn't we remove them? (e.g. update `rebuild.sh` to not build them anymore)?

Reply:
yup. That'll come as a new PR I guess. Probably I should have done this first, but :shrug:

Reply:
I have this semi-done is my working copy. I can post a draft.

Reply:
For the record, I looked into it. There needs to be done significant work to drop 4.14 guests. There are various places where we assume 4.14.